src/components/SpaceShip/SpaceShip.tsx

```diff
@@ -1,13 +1,9 @@
-interface SpaceShipProps {
```

Comment:
We're not actually using this styling or prop for hasBanner anymore so removed the prop/classname